test/commands/context.test.ts

```diff
@@ -6,14 +6,26 @@ import TestHelper from '../testHelper';
 const testHelper = new TestHelper();
 
 describe('config', () => {
-  beforeEach(() => {
+  beforeAll(() => {
     testHelper.createDummyContextFile();
   });
 
-  afterEach(() => {
+  afterAll(() => {
     testHelper.deleteDummyContextFile();
   });
 
+  describe('config:context:current', () => {
```

Comment:
this should fix random:
```
EPERM: operation not permitted, open 'D:\a\cli\cli\test.asyncapi'

      27 |
      28 |   createDummyContextFile(): void {
    > 29 |     writeFileSync(DEFAULT_CONTEXT_FILE_PATH, JSON.stringify(this._context), { encoding: 'utf-8' });
         |                  ^
      30 |   }
      31 |   
      32 |   deleteDummyContextFile(): void {
```

because as I can see, the context tests do not really need fresh context file for each test case and we can have it create just once